shared/chat/conversation/bottom-banner/index.tsx

```diff
@@ -72,10 +74,24 @@ const BrokenTrackerBanner = ({users, onClick}: BrokenTrackerProps) =>
     </BannerBox>
   )
 
-const InviteBanner = ({users}: InviteProps) => (
-  <BannerBox color={globalColors.blue}>
-    <BannerText>Your messages to {users.join(' & ')} will unlock when they join Keybase.</BannerText>
-  </BannerBox>
-)
+const InviteBanner = ({users, inviteEnabled, onShareClick}: InviteProps) =>
+  inviteEnabled && users.length === 1 && users[0].endsWith('@phone') ? (
```

Comment:
Why limit to just one other participant?

Reply:
gonna ask design about this